content/medo_manual/werken_met_medewerkers.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Werken met medewerkers
+sort: 400
+---
+
+* TOC
+{:toc}
+
+## Medewerkers
+
+Coördinatoren kunnen de medewerkers die bij een project horen beheren. Op deze
+pagina bevindt zich een tabel met de medewerkers. Door op een rij te klikken
+kunt u de rol van een medewerker op het project aanpassen.
```

Comment:
kunt u de instellingen van de medewerker aanpassen of de medewerker verwijderen

(je kunt dankzij jouw PR meer dan de rol aanpassen)